INSTALL

```diff
@@ -0,0 +1,69 @@
+Deploying UADK
+
+UADK is a framekwork for user application to access hardware, and it depends
+UACCE in kernel.
+
+
+1. Kernel Requirement
+
+User needs to make sure that UACCE is already supported in Linux kernel. The
+kernel version should be at least v5.9.
```

Comment:
In fact, we need a SVA kernel, so v5.9 is not enough :)

Reply:
fixed